pypeit/core/extract.py

```diff
@@ -305,6 +305,7 @@ def extract_optimal(sciimg,ivar, mask, waveimg, skyimg, rn2_img, oprof, box_radi
         specobj.optimal['WAVE'] = junk
         specobj.optimal['COUNTS'] = junk
         specobj.optimal['COUNTS_IVAR'] = junk
+        specobj.optimal['COUNTS_SIG'] = junk
```

Comment:
Be sure to update the data model for extraction.

Reply:
Yep, it's been updated. Although that is not documented anywhere anyway.